doc/source/devel/hydro_infrastructure.rst

```diff
@@ -820,45 +799,45 @@ This is a template class that provides the following features at compile
 time:
 
     * a lookup table (LUT) that maps the names of components of a subset
-      of the actively advected quantities defined in ``FIELD_TABLE`` to
-      unique, contiguous indices.
+      of the actively advected integration quantities defined in
+      ``FIELD_TABLE`` to unique, contiguous indices.
 
-    * the number of quantity components included in the table
+    * the number of integration quantity components included in the table
 
-    * a way to iterate over just the conserved quantities or specific
+    * a way to iterate over just the conserved or specific integration
       quantities values that are stored in an array using these mapping
 
-    * a way to query which of the actively advected quantities in
-      FIELD_TABLE are not included in the LUT
+    * a way to query which of the actively advected integration quantities
+      in FIELD_TABLE are not included in the LUT
 
 These feature are provided via the definition of publicly accessible
 integer constants in every specialization of the template class. All
 specializations have:
 
-    * a constant called ``NEQ`` equal to the number of quantity components
-      included in the lookup table
+    * a constant called ``NEQ`` equal to the number of integration quantity
```

Comment:
Just out of curiosity, is there is a reason to use NEQ for the name of this constant (rather than  something more descriptive as generally used elsewhere in the design)?

Reply:
That's a good point. Unfortunately, changing that name in this PR would introduce some a merge conflict with PR #113.

Therefore, I just added a commit to PR #113 to address this (I changed `NEQ` to `num_entries`). Does that sound reasonable to you? If you prefer, I can drop that commit and move those changes to this PR.

Reply:
Making the change in PS #113 is perfect, thanks!